Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/DevicesStepDefinitions.cs

```diff
@@ -156,7 +156,7 @@ public async Task WhenDeviceSendsADeleteRequestToTheDeviceIdEndpointWithTheDevic
         _responseContext.WhenResponse = await client.Devices.DeleteDevice(deviceId);
     }
 
-    [When($"{RegexFor.SINGLE_THING} sends a DELETE request to the /Devices/{{id}} endpoint with a non existent id")]
+    [When(@"{word} sends a DELETE request to the \/Devices\/\{id\} endpoint with a non existent id")]
```

Comment:
You sometimes don't use `^` and `$` in the regular expressions. Is there a specific reason for that?

Reply:
I read about the Cucumber expressions shortly after sending my review. They sound pretty interesting. But I think we use too many `/`s and `{`s for it to be readable, which is why I would probably rather use regex. But I'd suggest that we discuss that in tomorrow's daily and vote on what to do.